doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3685,6 +3685,10 @@ Setting implicit automation tactics
       time the term argument of a tactic has one of its holes not fully
       resolved.
 
+   .. deprecated:: 8.9
+      This command is deprecated. Use :ref:`typeclasses <typeclasses>` or
+      :ref:`tactics-in-terms <tactics-in-terms>`instead.
```

Comment:
This should be indented more `.. deprecated` should be at the same level as the preceding paragraph. In its current form, it is the cause of a Sphinx failure (see Travis).

Reply:
@Zimmi48 I've no idea about what I'm doing, let's hope it's working now.